apps/web/src/components/auth/forgot-form.tsx

```diff
@@ -0,0 +1,61 @@
+"use client";
+
+import { useState, useTransition } from "react";
+import { forgotPassword } from "@/actions/auth";
+import { forgotSchema } from "@/lib/schema/auth";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { Loader2Icon } from "lucide-react";
+import { useForm } from "react-hook-form";
+import type { z } from "zod";
+
+import { Button, Form, FormControl, FormDescription, FormField, FormItem, FormMessage, Input } from "@energyleaf/ui";
+import { useToast } from "@energyleaf/ui/hooks";
+
+export default function ForgotForm() {
+    const [loading, setLoading] = useState<boolean>(false);
```

Comment:
Besser mit [useTransition](https://react.dev/reference/react/useTransition) machen. Wir haben einige Beispiele, wie Server Funktionen aufgerufen werden.

https://nextjs.org/docs/app/building-your-application/data-fetching/server-actions-and-mutations#non-form-elements